ServiceFinder/ServiceFinder/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for ServiceFinder project.
+
+Generated by 'django-admin startproject' using Django 5.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-xjni&&k5_)8zqujz6y(kqw!8rajdyz4_y*ru*ml3=6qo_ht!uv'
```

Comment:
should this be here? 😳

Reply:
Found this resource for putting `SECRET_KEY `in `env`: [Link](https://stackoverflow.com/questions/15209978/where-to-store-secret-keys-django)